### PR TITLE
restrict size of notification container on small devices

### LINKED ISSUE #986: Edit target button not accessible on small device
**Describe the bug**
The button to open the dialog to change the tree target in your personal profile in covered by another layer on small devices, so it's not accessible any more:

Bildschirmfoto 2021-04-15 um 09 33 12

**Is it on a Production or Development Build?**

**To Reproduce**
1. Login and view your profile
2. Try to change your target
3. See error

**Expected behavior**
Open dialog to edit target.

**What actually happens**
Dialog won't open on small devices.

**Desktop (please complete the following information):**
- OS: latest MacOS
- Browser: latest Chrome, Safari emulating small devices

**Smartphone (please complete the following information):**
- Device: iPhone
- OS: latest iOS
- Browser: Safari

### PR DESCRIPTION
**Pull request by norbertschuler**
Fixes #986

Changes in this pull request:
- restrict size of notification container on small devices

I tried to limit the size of the notification container to max 15% of the screen size on smaller devices now, so it can not show big notifications any more, but it seems the notifications are shown anyway ignoring the size of the container:

Bildschirmfoto 2021-04-15 um 09 59 28

I am not sure if this work with all devices and thus solves the problem or creates a new one?

### CODE REVIEW

**Comment by sagararyal** [comment]
@sarveshpro can you check why notification container, even when deactivated is still interfering with the UI?

**Comment by sarveshpro** [comment]
The container had fixed height that's why it was interfering, made it `fit-content` now, looks good in Chrome on windows and android. Please test on Safari before merging(have faced problems with fit-content on Safari)
image

**Comment by norbertschuler** [comment]
This looks like the right solution. Also working on Safari in normal and responsive mode. Thank you. I can not approve as I created this PR :-) Please approve and merge.

**Review by harshvitra — APPROVED** [review]

**Pull request merged**
